package.json

```diff
@@ -31,6 +31,9 @@
     "win": {
       "target": "NSIS",
       "icon": "build/icon.ico"
+    },
+    "linux": {
+      "target": "deb"
```

Comment:
This section tells the `electron-builder build` command to produce a debian package as well as dmg and exe.
Ran it through the docker image: https://hub.docker.com/r/electronuserland/electron-builder/
And it produced a debian package that I could install on Ubuntu (16.04)